CMakeLists.txt

```diff
@@ -234,7 +249,7 @@ if(IPO_SUPPORTED)
 endif()
 
 if(NOT MSVC)
-    target_link_libraries(flint m)
+    target_link_libraries(flint PRIVATE m)
```

Comment:
This should have been PUBLIC. sorry about that

Reply:
OK, I'll try this. 

By the way, is it possible to build in parallel? It seems that `make -j8` makes no difference to `make -j1`.

Reply:
You can build and test in parallel. I don't know why `make -j8` doesn't make any difference

Reply:
For testing in parallel it seems that I need `ARGS="-j 4" make test` rather that `make -j4 test`. (That's what I gather from reading the Makefile).

Reply:
You could do `ctest -j4`

Reply:
> You can build and test in parallel. I don't know why `make -j8` doesn't make any difference

it's because cmake needs to know that its builder will be a "Unix make", so this has to be passed as a `-G"..."` argument. I've made this change in the README.